models/widget_token.go

```diff
@@ -97,10 +96,6 @@ func (m *WidgetToken) ContextValidate(ctx context.Context, formats strfmt.Regist
 
 func (m *WidgetToken) contextValidateInheritList(ctx context.Context, formats strfmt.Registry) error {
 
-	if err := validate.ReadOnly(ctx, "inheritList", "body", []*WidgetTokenInheritance(m.InheritList)); err != nil {
-		return err
-	}
```

Comment:
I don't think this needs to be removed, right?

Reply:
No, this could be something to do with the widget token issue we were trying to fix. I compared this file with our stash repo, where the fix is merged, and we are good to remove this. Please refer to https://stash.logicmonitor.com/projects/TEAMINTEGRATIONS/repos/terraform-integration/browse/terraform-provider-logicmonitor/models/widget_token.go?until=01f0fd82f8999f157bce3dbe128f36eeaa72e276&untilPath=terraform-provider-logicmonitor%2Fmodels%2Fwidget_token.go